### PR TITLE
MAT-7050: Add null check on groups for cypress workflow

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-7050](https://jira.cms.gov/browse/MAT-7050)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/services/HumanReadableService.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ String getCmsId(Measure measure) {
   HumanReadableMeasureInformationModel buildMeasureInfo(Measure measure) {
     boolean patientBased = false;
     String measureScoring = "";
-    if (!measure.getGroups().isEmpty()) {
+    if (measure.getGroups() == null || !measure.getGroups().isEmpty()) {
```
Comment (sb-prateekkeerthi) [inline src/main/java/gov/cms/madie/services/HumanReadableService.java]: can we do CollectionUtils.isEmpty(measure.getGroups()) I think it will check both null and empty condition

**Review by sb-prateekkeerthi — DISMISSED** [review]

File: src/main/java/gov/cms/madie/services/HumanReadableService.java (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/main/java/gov/cms/madie/services/HumanReadableService.java]: also shouldn't the condition have negation in measure.getGroups() == null ?

**Review by sb-prateekkeerthi — APPROVED** [review]
